samples/test/features/basic-auth.feature

```diff
@@ -17,4 +17,11 @@ Feature: Direct Auth Basic Login with Password Factor
     #   When User enters username into the form
     #   And User enters password into the form
     #   And User submits the form
-    #   Then User can verify their profile data
\ No newline at end of file
+    #   Then User can verify their profile data
+
+    Scenario: Mary doesn't know her username
+      Given Mary navigates to Login with Username and Password
+      When she fills in her incorrect username
+      And she fills in her password
+      And she submits the Login form
+      Then she should see a message on the Login form "There is no account with the Username Mory."
```

Comment:
I believe the actual message returned from server is

`You do not have permission to perform the requested action.`

Test is failing on Bacon:

```
Expected: "There is no account with the Username Mory."
Received: "You do not have permission to perform the requested action."
```

Let's use the actual server string here. (I'll get the cucumber spec updated)